config/routes.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  resources :post
+  # get 'post/index'
+  # get 'post/new'
+  # get 'post/show'
+  # get 'post/create'
+
+  get 'user/index'
+  get 'user/new'
+  get 'user/show'
+  get 'user/create'
```

Comment:
- [ ] Let's not expose unnecessary endpoints by using `resources :posts`  as this will expose some endpoints that you do not have in your controller actions yet. Let's create only the endpoints with the actions you have in your controller.

- [ ] Let's nest the posts endpoint inside the users' endpoints so that from a user show page, we can get to their posts index.

- [ ] Kindly remove the commented codes to achieve a more clean codebase

```suggestion
  resources :users, only: [:index, :show, :create, :new] do
    resources :posts, only: [:index, :show, :create, :new]
  end
```